caluma/form/serializers.py

```diff
@@ -258,6 +260,50 @@ def update(self, instance, validated_data):
         return instance
 
 
+class SaveDynamicQuestionOptionsMixin(object):
```

Comment:
Is this intended and what might the use cases be?

Reply:
Just to provide some context: We were wondering if the dynamic question types really should support additional, statically configured options.

Reply:
Ha! I really needed this couple of days off, things look much clearer now ;)

The options are now fetched every time the question is accessed. No options are created for those questions in the DB.